pyproject.toml

```diff
@@ -1,4 +1,8 @@
 [tool.astropy-bot]
+[build-system]
```

Comment:
You didn't restore everything here.
```suggestion
[tool.astropy-bot.changelog_checker]
filename = "CHANGES.rst"

[build-system]
```

Reply:
Hmm. Is that the same with `astropy-bot`? This is what's set up in the config. Or am I totally misunderstanding the issue here?

Reply:
`tool.astropy-bot.changelog_checker` is a subsection under `tool.astropy-bot`.